src/summary/history/pagination.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect } from "react";
+import styles from "./notes-card-style.css";
+
+export default function Pagination(props: PaginationProps) {
+  const [emptyResult, setEmptyResult] = useState(false);
+  const [pageNumbers, setPageNumbers] = useState(1);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalData, setTotalData] = useState(0);
+  const [resultsPerPage, setResultsPerPage] = useState(null);
+  const [showNextButton, setShowNextButton] = useState(false);
+  const [showPreviousButton, setShowPreviousButton] = useState(false);
+  let indexOfLastResult: number = currentPage * resultsPerPage;
+  let indexOfFirstResult: number = indexOfLastResult - resultsPerPage;
+  useEffect(() => {
+    currentPage * resultsPerPage > totalData
+      ? setShowNextButton(false)
+      : setShowNextButton(true);
+    currentPage !== 1
+      ? setShowPreviousButton(true)
+      : setShowPreviousButton(false);
+    paginate();
+  }, [totalData, currentPage, resultsPerPage]);
+  useEffect(() => {
+    setTotalData(props.totalData);
+    setResultsPerPage(props.resultsPerPage);
+    setPageNumbers(Math.ceil(props.totalData / props.resultsPerPage));
+  }, [props]);
+  return (
+    <table style={{ width: "100%" }}>
```

Comment:
why are you using a table here?

Reply:
most of the level two components have been made using tables,